### ISSUE TITLE
Github action for auto bump packages

### ISSUE THREAD

**Issue by ericdallo** [ryantm/nixpkgs-update#249]
Is there a Github Action for auto bump packages in nixpkgs like there is for [brew](https://github.com/mislav/bump-homebrew-formula-action)?

With that, package maintainers could setup the `nixpkgs-update-action` in their repos and the action would automatically open an PR with the given github token.
WDYT?

**Comment by ryantm** [comment]
There is not currently. Sounds like a nice idea.

**Comment by ericdallo** [comment]
@ryantm I could help on that, but one thing that may be hard is to get current repo previous version since the action would be triggered after a new release is pushed.
Is there any way to use `nixpkgs-update` and do not pass the current version, just the new?
like: `nixpkgs-update postman 2.0.0`

**Comment by davidak** [comment]
this would add more dependency on github

**Comment by ryantm** [comment]
@ericdallo Yeah, it would be good to support that style of updates. Unfortunately, it doesn't currently.

**Comment by Synthetica9** [comment]
Good idea, might take a stab at this later.

@davidak: the action could live in a separate repo, so I don't see how.

**Comment by davidak** [comment]
@Synthetica9 it doesn't matter in which repo it is configured.

if we want to migrate away from github, it becomes more complicated the more features we use

when it is run on the community infrastructure, github is not a dependency for it

**Comment by ericdallo** [comment]
@davidak when/if we migrate to github I bet we will find/replace with other solutions, for now, IMO this looks a good idea for what we have now

**Comment by Synthetica9** [comment]
@davidak Say we migrate to say... Gitlab. 

In that case, we could simply update the action to open a PR to the new Gitlab repo instead of the old Github one. The action would still need to be a Github action, because it doesn't matter what platform _we're_ on as Nix, but what platform the party using this action is on. The converse also applies (though I don't know if Gitlab has an equivalent to Github actions).
